src/LayerEditor/ui/panels/surfaces.py

```diff
@@ -1,181 +1,78 @@
 import PyQt5.QtWidgets as QtWidgets
```

Comment:
This file has some interesting stuff. 5 SurfFormData was merged with SurfaceItem.